fennel/featuresets/featureset.py

```diff
@@ -576,6 +582,60 @@ def _get_extractors(self) -> List[Extractor]:
         return extractors
 
     def _validate(self):
+        cls_module = inspect.getmodule(self.__fennel_original_cls__)
+        if cls_module is not None and hasattr(
+            cls_module, FENNEL_GEN_CODE_MARKER
```

Comment:
What does this do?

Reply:
This is an interesting issue I encountered. So basically we ship code. But when the shipped code runs, since it runs under exec mode, all types are strings. Hence for the shipping code typing validation breaks. However we dont need to run validation once it has already run by the user, so we disable the validation when executing shipped code

Reply:
Ahh I see